### PR TITLE
Created ReadME with the info. about the project

### LINKED ISSUE #1: Add a world class developer friendly readme for this project
The readme for any project is as good as the project itself. I was looking through different Readme files and I tried compiling something as well. The one I came up with is something like this:

**Table of Contents**

- Name Inspirations
- Demo
- Use Cases
- Working
- Development
   - System Requirements
   - Installation
   - Testing
- Contributing
- Thanks

Currently, the readme this project has, is created by the Create React App boilerplate. I have seen some good ideas coming up here: [matiassingers/awesome-readme: A curated list of awesome READMEs](https://github.com/matiassingers/awesome-readme), where we can get inspiration from.

We need something that is not cluttered as well as looks professional and decent. 😊

### PR DESCRIPTION
**Pull request by AnjaliSharma1234**
## Issue that this pull request solves

Closes: #1  (issue number)

## Proposed changes

Created a new ReadME file with information about the project, mentors, admin and the motivation behind the project.

## Types of changes

_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation update (Documentation content changed)
- [ ] Other (please describe): 

## Checklist

_Put an `x` in the boxes that apply_

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings

### CODE REVIEW

File: ReadME.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+
+## Installation Instructions
+
+
+
+## Contribution Guidelines
+
+[![GitHub issues](https://img.shields.io/github/issues/CatsInTech/Rezume?logo=github)](https://github.com/codeuino/Rezume/issues) ![GitHub pull requests](https://img.shields.io/github/issues-pr-raw/CatsInTech/Rezume?logo=git&logoColor=white) ![GitHub contributors](https://img.shields.io/github/contributors/CatsInTech/Rezume?logo=github)  [![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg?style=flat&logo=git&logoColor=white)](https://github.com/CatsInTech) 
+
+ [![GitHub last commit](https://img.shields.io/github/last-commit/CatsInTech/Rezume?logo=github)](https://github.com/CatsInTech) [![CatsInTech](https://img.shields.io/badge/Author-@praveenscience-gray.svg?colorA=gray&colorB=dodgerblue&logo=github)](https://github.com/CatsInTech/)
```
Comment (praveenscience) [inline ReadME.md]: Get rid of the space before this.

File: ReadME.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+- When you make very very minor changes to a PR of yours (like for example fixing a text in button, minor changes requested by reviewers) make sure you squash your commits afterwards so that you don't have an absurd number of commits for a very small fix. (Learn how to squash at [here](https://davidwalsh.name/squash-commits-git))
+- When you're submitting a PR for a UI-related issue, it would be really awesome if you add a screenshot of your change or a link to a deployment where it can be tested out along with your PR. It makes it very easy for the reviewers and you'll also get reviews quicker.
+- Please follow the [PR Template](https://github.com/CatsInTech/Rezume/blob/master/CONTRIBUTING.md) to create the PR.
+- Always create PR to `develop` branch.
+
+
+> **Note:** Are you an **absolute beginner?** Please [check the issues](https://github.com/CatsInTech/Rezume/issues) labeled with `first-timers-only`. These issues might _let your Open Source journey begin_ - **Welcome onboard!**
```
Comment (praveenscience) [inline ReadME.md]: We can start with with `beginners` label itself. And a couple of grammatical fixes below:

```suggestion
> **Note:** Are you an **absolute beginner**? Please [check the issues](https://github.com/CatsInTech/Rezume/issues) labeled with `beginners`. These issues might _let your Open Source journey begin_ - **Welcome on-board!**
```

File: ReadME.md [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+[![GSSoC](https://raw.githubusercontent.com/GirlScriptSummerOfCode/MentorshipProgram/master/GSsoc%20Type%20Logo%20Black.png)](https://www.gssoc.tech/)
+
+**1. How to start as a beginner so as to contribute to the project?**
+*There are many beginner level issues to begin with, which include documentation, development and design. Also, feel free to make issues for features as well as for reporting bugs. Participants can claim an issue by commenting that they are interested in it. We are also open to new ideas and suggestions.*
+
+**2. How to communicate with us?**
+*All communication will happen on the Slack channel for this particular projects. Mentors can also be reached through GitHub. Avoid DMs and unnecessary mentions on slack and GitHub both. *Checkout [this](https://github.com/CatsInTech/Rezume/issues) issue for more.**
+
+**3. What is the deadline for any task submission?**
+
```
Comment (praveenscience) [inline ReadME.md]: Keep the spaces consistent. Either use a space or remove it everywhere.

File: ReadME.md [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+| [@priyadharshinir1894](https://github.com/priyadharshinir1894) | Full Stack + Cloud         |
+| [@vermakhushboo](https://github.com/vermakhushboo) | Full Stack         |
+| [@PrathibhaS](https://github.com/PrathibhaS) | Backend |
+| [@AnjaliSharma1234](https://github.com/AnjaliSharma1234) | Frontend + Documentation |
+
+## Project Admin
+
+[![Relative date](https://img.shields.io/date/1577392258?color=important&label=started&logo=github)](https://github.com/praveenscience/) [![Maintenance](https://img.shields.io/maintenance/yes/2020?color=green&logo=github)](https://github.com/praveenscience/)
+
+
```
Comment (praveenscience) [inline ReadME.md]: Max 1 line space, not more than that.

File: ReadME.md [hunk 5]
```diff
@@ -0,0 +1,98 @@
...
+> ***Need help?***
+***Feel free to contact me at [catsintech@praveen.science](mailto: catsintech@praveen.science?Subject=Cats in Tech Project)***
+
+[![GitHub followers](https://img.shields.io/github/followers/praveenscience.svg?label=Follow%20@praveenscience&style=social)](https://github.com/praveenscience/) [![Twitter Follow](https://img.shields.io/twitter/follow/praveenscience?style=social)](https://twitter.com/praveenscience) 
+
+---
+
+[![built with love](https://forthebadge.com/images/badges/built-with-love.svg)](https://github.com/AnjaliSharma1234/)
+
+
```
Comment (praveenscience) [inline ReadME.md]: Get rid of trailing spaces on each lines and trailing lines at the end of document.

**Comment by AnjaliSharma1234** [comment]
I have made the changes and pushed the updated ReadME. Can you review that @praveenscience?

File: ReadME.md [hunk 6]
```diff
@@ -0,0 +1,86 @@
...
+- To let beginners venture into the domain of Web Development
+- To facilitate students and working professionals to create decent Resumes by focusing on the content rather than the styling and other UI features
+
+## Tech Stack 
+
+- **Frontend:** HTML5, CSS3, JavaScript, Bootstrap, ReactJS 
+- **Backend:** NodeJS
+
+## Installation Instructions
+
```
Comment (praveenscience) [inline ReadME.md]: Replace the above with something like this. Check your email for the complete instructions:

```
## Getting Started

### System Requirements

More in your email.
```

Comment (AnjaliSharma1234) [reply ReadME.md]: I have updated the document, kindly check @praveenscience

File: ReadME.md [hunk 7]
```diff
@@ -0,0 +1,137 @@
+# Welcome to Rezume - a free Resume Generator
+
+[![first-timers-only](https://img.shields.io/badge/first--timers--only-friendly-tomato.svg?style=flat&logo=git)](https://github.com/CatsInTech/Rezume/issues) [![GitHub stars](https://img.shields.io/github/stars/CatsInTech/Rezume.svg?logo=github)](https://github.com/CatsInTech/Rezume/stargazers) [![GitHub forks](https://img.shields.io/github/forks/CatsInTech/Rezume.svg?logo=github&color=teal)](https://github.com/CatsInTech/Rezume/network) [![GitHub top language](https://img.shields.io/github/languages/top/CatsInTech/Rezume?color=yellow&logo=javascript)](https://github.com/vinitshahdeo/Water-Monitoring-System)
```
Comment (praveenscience) [inline ReadME.md]: Why is this coming here?
```
https://github.com/vinitshahdeo/Water-Monitoring-System
```

File: ReadME.md [hunk 8]
```diff
@@ -0,0 +1,137 @@
+# Welcome to Rezume - a free Resume Generator
+
+[![first-timers-only](https://img.shields.io/badge/first--timers--only-friendly-tomato.svg?style=flat&logo=git)](https://github.com/CatsInTech/Rezume/issues) [![GitHub stars](https://img.shields.io/github/stars/CatsInTech/Rezume.svg?logo=github)](https://github.com/CatsInTech/Rezume/stargazers) [![GitHub forks](https://img.shields.io/github/forks/CatsInTech/Rezume.svg?logo=github&color=teal)](https://github.com/CatsInTech/Rezume/network) [![GitHub top language](https://img.shields.io/github/languages/top/CatsInTech/Rezume?color=yellow&logo=javascript)](https://github.com/vinitshahdeo/Water-Monitoring-System) 
+
+
```
Comment (praveenscience) [inline ReadME.md]: Spaaacesss!!! 😅

File: ReadME.md [hunk 9]
```diff
@@ -0,0 +1,86 @@
...
+**2. How to communicate with us?**
+*All communication will happen on the Slack channel for this particular projects. Mentors can also be reached through GitHub. Avoid DMs and unnecessary mentions on slack and GitHub both. *Checkout [this](https://github.com/CatsInTech/Rezume/issues) issue for more.**
+
+**3. What is the deadline for any task submission?**
+*Mentors can assign issues to participants and also mention a deadline before which the issue is to be resolved. They can reassign issues in case of any delay, so be active and complete your tasks within the negotiated duration.*
+
+## Mentors
+
+ [![Open Source Love](https://badges.frapsoft.com/os/v2/open-source.svg?v=103)](https://github.com/CatsInTech/Rezume) ![GitHub pull requests](https://img.shields.io/github/issues-pr-closed-raw/CatsInTech/Rezume?logo=git&logoColor=white)
```
[the remark this answers is not in the document]
Comment (praveenscience) [reply ReadME.md]: @AnjaliSharma1234 You forgot to add this.

File: ReadME.md [hunk 10]
```diff
@@ -0,0 +1,86 @@
...
+## Project Admin
+
+[![Relative date](https://img.shields.io/date/1577392258?color=important&label=started&logo=github)](https://github.com/praveenscience/) [![Maintenance](https://img.shields.io/maintenance/yes/2020?color=green&logo=github)](https://github.com/praveenscience/)
+
+| <a href="https://praveen.science/"><img src="https://i.imgur.com/pQkeC0B.png" width=150px height=120px /></a> |
+| :----------------------------------------------------------: |
+| **[Praveen Kumar Purushothaman](https://praveen.science/)**  |
+
+> ***Need help?***
+***Feel free to contact me at [catsintech@praveen.science](mailto: catsintech@praveen.science?Subject=Cats in Tech Project)***
```
[the remark this answers is not in the document]
Comment (praveenscience) [reply ReadME.md]: @AnjaliSharma1234 You forgot to add this.

**Comment by AnjaliSharma1234** [comment]
Now, I have updated everything. Please review @praveenscience

File: ReadME.md [hunk 11]
```diff
@@ -0,0 +1,136 @@
...
+| [@priyadharshinir1894](https://github.com/priyadharshinir1894) | Full Stack + Cloud         |
+| [@vermakhushboo](https://github.com/vermakhushboo) | Full Stack         |
+| [@PrathibhaS](https://github.com/PrathibhaS) | Backend |
+| [@AnjaliSharma1234](https://github.com/AnjaliSharma1234) | Frontend + Documentation |
+
+## Project Admin
+
+[![Relative date](https://img.shields.io/date/1577392258?color=important&label=started&logo=github)](https://github.com/praveenscience/) [![Maintenance](https://img.shields.io/maintenance/yes/2020?color=green&logo=github)](https://github.com/praveenscience/)
+
+| <a href="https://praveen.science/"><img src="https://i.imgur.com/pQkeC0B.png" width=150px height=120px /></a> |
```
Comment (praveenscience) [inline ReadME.md]: Any issues with using Markdown here? ✨

File: ReadME.md [hunk 12]
```diff
@@ -0,0 +1,136 @@
...
+| :----------------------------------------------------------: |
+| **[Praveen Kumar Purushothaman](https://praveen.science/)**  |
+
+>**Need help?**
+Feel free to contact me at [catsintech@praveen.science](https://praveen.science/).
+
```
Comment (praveenscience) [inline ReadME.md]: This is definitely misleading. 😔 Let's use this way! 😅 And yea. No empty lines. 😅
```suggestion
Feel free to contact me at [catsintech@praveen.science](mailto:catsintech@praveen.science).
```

Comment (AnjaliSharma1234) [reply ReadME.md]: I think this "mailto" was not working as already showed it to you. Do you still want to get it changed? @praveenscience

File: ReadME.md (hunk 11 above)
Comment (AnjaliSharma1234) [reply ReadME.md]: No, but I did not see any harm in using this :) Any issues with it though? @praveenscience

Comment (praveenscience) [reply ReadME.md]: Looks weird, that's all! 😅

**Comment by AnjaliSharma1234** [comment]
Thank you @vermakhushboo @praveenscience

**Pull request merged**